services/app-web/src/pages/StateSubmission/RateDetails/V2/RateDetailsV2.tsx

```diff
@@ -539,6 +578,66 @@ const RateDetailsV2 = ({
                                             </>
                                         )}
                                     </FieldArray>
+                                    <SectionCard>
```

Comment:
From the [figma](https://www.figma.com/file/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?type=design&node-id=12176-104410&mode=design&t=rf1bdhrjcAryfOlh-0) link, it looks like each rate has its own actuary com preference.